src/demo/coupons.js

```diff
@@ -12,25 +12,20 @@ if (configuration.NODE_ENV === 'development') {
   firstCouponHasError = true;
 }
 
-const coupons = {
-  count: couponsCount,
-  num_pages: Math.ceil(couponsCount / 10),
-  current_page: 1,
-  results: [...Array(couponsCount)].map((_, index) => {
-    const validFromDate = faker.date.past();
-    const totalEnrollments = faker.random.number({ min: 1, max: 50 });
-    return {
-      id: index,
-      title: faker.random.words(),
-      start_date: validFromDate.toISOString(),
-      end_date: faker.date.future(null, validFromDate).toISOString(),
-      num_unassigned: faker.random.number({ min: 1, max: 20 }),
-      num_uses: faker.random.number({ min: 1, max: totalEnrollments }),
-      max_uses: faker.random.boolean() ? totalEnrollments : null,
-      has_error: false,
-    };
-  }),
-};
+const coupons = [...Array(couponsCount)].map((_, index) => {
+  const validFromDate = faker.date.past();
+  const totalEnrollments = faker.random.number({ min: 1, max: 50 });
+  return {
+    id: index,
+    title: faker.lorem.words(faker.random.number({ min: 1, max: 3 })).toUpperCase(),
```

Comment:
Instead of generating random words from faker, which still may not be realistic, can we create an array of realistic coupon names and the `title` pulling in a random one for the title?

It seems like some of the coupon names we have are titled like `10OFF`, or `100OFF`. Using titles that reflect actual names may be a good idea here. Thoughts?